docs/source/rbac/tech-implementation.md

```diff
@@ -43,7 +43,7 @@ Prior to 3.0, tokens stored _roles_,
 which meant their scopes were resolved on each request.
 :::
 
-API tokens grant access to JupyterHub's APIs. The Role Based Access Control (RBAC) framework allows for the requesting of tokens with specific permissions.
+API tokens grant access to JupyterHub's APIs. The [RBAC framework](https://jupyterhub.readthedocs.io/en/stable/rbac/upgrade.html) allows for requesting tokens with specific permissions.
```

Comment:
This should be a local reference. You can use the reference `(./index.md)`.

Reply:
i have messed up again sir please give me time please

Reply:
sir @minrk  this should be the local reference  https://github.com/jupyterhub/jupyterhub/blob/main/docs/source/rbac/index.md i guess?